pkg/engine/const.go

```diff
@@ -110,6 +110,8 @@ const (
 	kubernetesDockerMonitorSystemdService    = "k8s/cloud-init/artifacts/docker-monitor.service"
 	labelNodesScript                         = "k8s/cloud-init/artifacts/label-nodes.sh"
 	labelNodesSystemdService                 = "k8s/cloud-init/artifacts/label-nodes.service"
+	untaintNodesScript                       = "k8s/cloud-init/artifacts/untaint-nodes.sh"
```

Comment:
I think a better antonym for "taint" is "cleanse." I insist that we use this verb everywhere from now on. 😜